docs/markdown/iOS/sheet.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Sheet
```

Comment:
It looks like we use a different title for the equivalent doc on Android. Is this intentional?